bamazonCustomer.js

```diff
@@ -0,0 +1,76 @@
+var mysql = require("mysql");
+var inquirer = require("inquirer");
+
+var connection = mysql.createConnection({
+  host: "localhost",
+  port: 3306,
+
+  user: "root",
+
+  password: "Oak04k88",
+  database: "bamazon_db"
+});
+
+connection.connect(function(err) {
+  if (err) throw err;
+  beginOrder();
+});
+
+function beginOrder() {
+  connection.query("SELECT * FROM products", function(err, res) {
+    if (err) {
+    	throw err;
+    }
+
+    for (var i = 0; i < res.length; i++) {
+      console.log(
+        "ID: " +
+          res[i].item_id +
+          " || Product: " +
+          res[i].product_name +
+          " || Price: " +
+          res[i].price
+      );
+    }
+
+    inquirer
+    .prompt([{
+      name: "idInput",
+      type: "input",
+      message: "enter product ID",
+      },
+      {
+      name: "order",
+      type: "input",
+      message: "enter product quantity",
+      }
+      ])
+    .then(function(answer) {
```

Comment:
I think here you should have some type and value checking to ensure that this is indeed a product name and quantity that can be used as input for a database query.